tests/test_index.py

```diff
@@ -884,3 +884,63 @@ def test_date_btree(self, reset_db: None) -> None:
             start + datetime.timedelta(days=random.randint(0, int(delta_days))) for _ in range(self.BTREE_TEST_NUM_ROWS)
         ]
         self.run_btree_test(data, pxt.Date)
+
+    @pxt.udf
+    @staticmethod
+    def dummy_embedding(text: str, n: int) -> pxt.Array[(None,), np.float32]:
```

Comment:
Do we have test coverage of the packager code too? (i.e., is that coverage implicit in the existing packager tests?)

Reply:
we do now